packages/bpk-tokens/formatters/react-native-value-template.js

```diff
@@ -14,9 +14,18 @@ export default function(value, type) {
       formattedValue = Number.isNaN(parsedDuration) ? value : parsedDuration;
       break;
     }
-    case 'string':
+    case 'string': {
       formattedValue = `"${value.replace(/"/g, '\\"')}"`;
       break;
+    }
+    case 'semantic': {
+      const parsedValues = Object.keys(value).reduce((parsed, tokenKey) => {
+        const { value: currValue, type: currType } = value[tokenKey];
+        return `${parsed} ${tokenKey}: ${formatValue(currValue, currType)},\n`;
+      }, '');
+      formattedValue = `{\n${parsedValues}};`;
```

Comment:
The `;` inside the string here is causing an exteraneous `;` to appear in the output files
```suggestion
      formattedValue = `{\n${parsedValues}}`;
```